### ISSUE TITLE
Button getting white border on focus in IE11 whereas in other browsers black border is being displayed when focused

### ISSUE THREAD

**Issue by shivasai09** [OfficeDev/office-ui-fabric-react#9971]
### Environment Information 
 
- __Package version(s)__: 6.186.1
- __Browser and OS versions__: IE11 and windows 10

### Please provide a reproduction of the bug in a codepen:
 

1. go to the link https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v6.186.1
2. download source code.
3. run the following commands respectively
         -     npm run _rushInstall
         -     npm run _rushBuild
         -     npm start
4. open the url http://localhost:4322/#/examples/button
5. observe that buttons have white border focus in IE11, whereas in other browsers such as chrome have black border focus

 
#### Actual behavior:
 focus border is white in color in IE11 for standard button, compound button, command bar button, Icon button etc, whereas in other browsers focus border is in black color
 
#### Expected behavior:
 focus border should be black in color, it should be similar to other browsers

#### Priorities and help requested:
 
Are you willing to submit a PR to fix?  No
 
Requested priority: Normal
 
Products/sites affected: 
M365adminUX

# images

standard button has white focus borer in IE11 whereas in chrome it's black in color
![image](https://user-images.githubusercontent.com/33802398/62041558-911c7a80-b219-11e9-9491-aab3bba8566c.png)

in chrome
![image](https://user-images.githubusercontent.com/33802398/62041584-a0032d00-b219-11e9-9475-eb3b448821ab.png)

same case with compound button in IE11
![image](https://user-images.githubusercontent.com/33802398/62041662-c88b2700-b219-11e9-9669-bfb78313ec0e.png)

in chrome
![image](https://user-images.githubusercontent.com/33802398/62041678-d2ad2580-b219-11e9-9662-bb7b458b9a18.png)

same case with command bar button in IE11
![image](https://user-images.githubusercontent.com/33802398/62041710-ed7f9a00-b219-11e9-9f02-c57dfecdb649.png)

in chrome
![image](https://user-images.githubusercontent.com/33802398/62041735-fbcdb600-b219-11e9-8a95-79762f651a5c.png)

also icon button has white border in IE11, it is not visible because background is also white
![image](https://user-images.githubusercontent.com/33802398/62042537-f2dde400-b21b-11e9-80f0-b57f94214fd3.png)

whereas in chrome it has black border
![image](https://user-images.githubusercontent.com/33802398/62042561-01c49680-b21c-11e9-8d0f-ebd24824f395.png)

These button are also used in commad bar component it is also effecting there.. (in IE11)
![image](https://user-images.githubusercontent.com/33802398/62042824-a47d1500-b21c-11e9-811b-e493070a0a30.png)

(in chrome)
![image](https://user-images.githubusercontent.com/33802398/62042690-4fd99a00-b21c-11e9-8e41-52c484defb51.png)

**Comment by micahgodbolt** [comment]
hmm, yup. I'm able to repro this in 6.197.0 as well

I think the answer might be around IE11 and outlines in pseudo elements. If you inspect you can see that the white is coming from the pseudo border (and is displaying properly) but the outline is not working (don't mind the line-throughs....not sure why its displaying like that. you can turn the border on/off and it works)

![image](https://user-images.githubusercontent.com/1434956/62314418-ca3a3200-b447-11e9-838b-c74ac941fe09.png)

**Comment by micahgodbolt** [comment]
Ahhhaaa! The outline works, but it's being clipped. Add overflow: visible to the button and the outline shows up.

![image](https://user-images.githubusercontent.com/1434956/62314648-59dfe080-b448-11e9-99c6-a38893f44b42.png)

**Comment by shivasai09** [comment]
hello @micahgodbolt if are using single button then we can add some css to the button from the app.
what happens to the other buttons such as commadbar buttons, there will be multiple types of buttons in the commadbar, so adding css for each button in the commandbar will be a overhead task.

can we just add overflow:visible in basebutton.styles.ts sent out the pr. is that a good idea?

**Comment by micahgodbolt** [comment]
yeah, sorry. I wasn't trying to provide a solution for you to apply. I was simply trying to diagnose the problem to make it easier for @marygans to work on this. Next question is, what can we do?

I don't like the idea of adding overflow visibile just to fix an IE11 bug. I"m 99% sure that a change of that scope would break a ton of partners. On the plus side, this only affects Fabric 6 (not 7), so any fix we do isn't something we have to carry on into production today.

**Comment by micahgodbolt** [comment]
Took a quick look to confirm, but buttons are by default overflow: visible.....at least in modern edge

![image](https://user-images.githubusercontent.com/1434956/62476839-9ff4b700-b75c-11e9-9437-d31ad92792aa.png)

But in IE11 they are defaulted to hidden. Problem is, we never saw the issue on our website because MWF normalizes buttons like this: 

![image](https://user-images.githubusercontent.com/1434956/62477403-c1a26e00-b75d-11e9-896f-da6f3ce9547c.png)

**Comment by micahgodbolt** [comment]
@shivasai09 since the problem here is a difference in default behavior between IE11 and modern Edge, the easiest solution on your side, with least chance of causing a break, is to add that above normalize code to your application (can even limit the loading to only IE11).

Another option is to use CSS normalize, which is used for this exact problem. https://github.com/necolas/normalize.css/blob/master/normalize.css#L176

**Comment by shivasai09** [comment]
@micahgodbolt  i will check it and let you know.
